examples/speech-recognition/README.md

```diff
@@ -16,6 +16,22 @@ limitations under the License.
 
 # Automatic Speech Recognition Examples
 
+## Table of Contents
+
+- [Automatic Speech Recognition with CTC](#connectionist-temporal-classification)
+	- [Single GPU example](#single-gpu-ctc)
+	- [Multi GPU example](#multi-gpu-ctc)
+	- [Examples](#examples-ctc)
+		- [TIMIT](#timit-ctc)
+		- [Librispeech](#librispeech-ctc)
+		- [Common Voice](#common-voice-ctc)
+		- [Multilingual Librispeech](#multilingual-librispeech-ctc)
+- [Automatic Speech Recognition with Sequence-to-Sequence](#sequence-to-sequence)
+	- [Whisper Model](#whisper-model)
+	- [Fine tuning](#single-hpu-whisper-training-with-seq2seq)
+	- [Inference](#single-gpu-seq2seq-inference)
```

Comment:
When you say the ones of Transformers example, which one specifically do you mean?
I tried to form a Table of contents, because the transformers repo had one, but may not match what I put here.

Reply:
I mean that the table of content contains several occurrences of "GPU" while it should probably be HPU

Reply:
I will fix this and run the "make style" that you requested.

Reply:
This is done.